_posts/2024-02-27-the-definitive-guide-2-jp.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "Japanese edition of Trino: The Definitive Guide"
+author: Yuya Ebihara
+image: /assets/ttdg2-jp-cover.jpg
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Do you know where the name 'Trino' comes from? It's actually a shortened form of
+'neutrino'. These fast and lightweight subatomic particles have recently made
+their way to Japan. You can now reserve your copy of the Japanese edition of
+[Trino: The Definitive Guide]({{site.url}}/trino-the-definitive-guide.html)!
+
+<!--more-->
+
+Today, we are happy to announce that the Japanese translation of the book
+[Trino: The Definitive Guide]({{site.url}}/trino-the-definitive-guide.html) is
+available for the communities all across Japan and far beyond. Hopefully it can
```

Comment:
Is it available now, or will it be available on March 19?

Reply:
Will be available on March 19. Only reservation is opened at this moment.

Reply:
I think this needs to be reworded - it is not available yet, and we should state the date it'll be accessible.